presage/src/manager/registered.rs

```diff
@@ -953,7 +967,12 @@ impl<S: Store> Manager<S, Registered> {
                         key: profile_key.derive_access_key().to_vec(),
                         certificate: sender_certificate.clone(),
                     });
-            recipients.push((member.uuid.into(), unidentified_access));
+            let include_pni_signature = true;
```

Comment:
This should come from the database, and PNI sigs should only be generated when necessary.

Reply:
This is now documented in the issue #252